frontend/src/lib/utils/authenticationStore.ts

```diff
@@ -72,7 +31,7 @@ export const loadLocalStorage = () => {
 };
 
 export const loadUserData = async () => {
-  const user = await me();
+  const user = (await me()) as User & {error: unknown};
```

Comment:
Should the types be applied to the `me` function instead so the casting here is unneeded (and applies to any other function that may use that function in the future)?

Reply:
Is it better now? I'm not sure what the actual type is but this is my best guess... The `{ data: T }` thing was copied from `getData.ts` because I thought the structure of the response would be similar, but clearly not :shrug: